### PR TITLE
Moved Login to service

### PR DESCRIPTION
**Pull request by Mnw2212**
1. There were errors in the services, methods were called on null objects . Rectified
2. Login worked without services, now integrated with services from Data/
3. Modified index.module.js to add the services from data and login

### CODE REVIEW

File: src/app/login/login.service.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+export class LoginServ{
```
Comment (skeep) [inline src/app/login/login.service.js]: just name it `Login`

File: src/app/login/login.service.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  constructor(FireData,FireConnect,$log,$location){
+    'ngInject';
+    this.clog = $log;
+    this.fireData = FireData;
+    this.fireConnect = FireConnect;
+    this.locate = $location;
```
Comment (skeep) [inline src/app/login/login.service.js]: `location` is noun, `locate` is verb. I would prefer `location`

File: src/app/login/login.service.js (hunk 1 above)
Comment (Mnw2212) [inline src/app/login/login.service.js]: so shall I name it as Auth.

**Comment by Mnw2212** [comment]
@skeep this can be merged

**Pull request merged**